generators/setup-docker/templates/fabric-docker/commands-generated.sh

```diff
@@ -51,76 +64,122 @@ function upgradeChaincode() {
     exit 1
   fi
 
-  <% chaincodes.forEach(function(chaincode) { %>
-  if [ "$chaincodeName" = "<%= chaincode.name %>" ]; then
-    <%- include('commands-generated-node-build.sh.ejs', {chaincode: chaincode}); -%>
-    <% chaincode.channel.orgs.forEach(function (org) {
-         org.peers.forEach(function (peer) {
-    %>
-    printHeadline "Installing '<%= chaincode.name %>' on <%= chaincode.channel.name %>/<%= org.name %>/<%= peer.name %>" "U1F60E"
-    <% if(!networkSettings.tls) { -%>
-    chaincodeInstall "cli.<%= org.domain %>" "<%= peer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "$version" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>"
-    <% } else { -%>
-    chaincodeInstallTls "cli.<%= org.domain %>" "<%= peer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "$version" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>" "crypto/orderer-tlscacerts/tlsca.<%= rootOrg.domain %>-cert.pem"
-    <% } -%>
-    <% })}) -%>
-
-    printItalics "Upgrading as '<%= chaincode.instantiatingOrg.name %>'. '<%= chaincode.name %>' on channel '<%= chaincode.channel.name %>'" "U1F618"
-    <% if(!networkSettings.tls) { -%>
-    chaincodeUpgrade "cli.<%= chaincode.instantiatingOrg.domain %>" "<%=  chaincode.instantiatingOrg.headPeer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "$version" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>" '<%- chaincode.init %>' "<%- chaincode.endorsement %>" "<%= chaincode.privateDataEnabled %>"
-    <% } else { -%>
-    chaincodeUpgradeTls "cli.<%= chaincode.instantiatingOrg.domain %>" "<%=  chaincode.instantiatingOrg.headPeer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "$version" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>" '<%- chaincode.init %>' "<%- chaincode.endorsement %>" "<%= chaincode.privateDataEnabled %>" "crypto/orderer-tlscacerts/tlsca.<%= rootOrg.domain %>-cert.pem"
-    <% } %>
-  fi
+  <% chaincodes.forEach(function(chaincode) { -%>
+    if [ "$chaincodeName" = "<%= chaincode.name %>" ]; then
+      <%- include('commands-generated-node-build.sh.ejs', {chaincode: chaincode}); -%>
+      <% chaincode.channel.orgs.forEach(function (org) { -%>
+        <% org.peers.forEach(function (peer) { -%>
+          printHeadline "Installing '<%= chaincode.name %>' on <%= chaincode.channel.name %>/<%= org.name %>/<%= peer.name %>" "U1F60E"
+          chaincodeInstall \
```

Comment:
same as above - i would reduce it. simply it's too much lines for commands-generated